### DISCUSSION TITLE
Copy actor name to clipboard

### DISCUSSION THREAD

**Discussion by C-O-D** [trakt/showly#773] [Ideas / Feature Requests]
Could you possibly add the option to copy the name of an actor to the clipboard with a long click, like it is possible with the movie/series title?

Best regards

**Comment by C-O-D** [comment]
Am I really in the right place to make a suggestion for a new feature?

**Reply by michaldrabik** [reply]
Yup - just cant answer everything 100% all the time : )

**Comment by michaldrabik** [comment]
Should not be too complicated will add it for my notes. By the way, if you are trying to copy name to search in Google for example you can already do it via actor's links menu (fyi)

**Reply by C-O-D** [reply]
I know, but this wish seemed easier to realize than the integration of a custom search engine that you can configure yourself - just for those cases in which the certainly already very rich offer of preconfigured search items is not sufficient.
Anyway, thanks for taking care of the matter...

**Comment by C-O-D** [comment]
Thanks very much for implementing 👍
